tests/wpunit/ConnectionRegistrationTest.php

```diff
@@ -317,7 +317,7 @@ static function () {
 		$this->assertQueryError( $response, $expected );
 
 		\wp_set_current_user( 1 );
-		$this->clearSchema();
+
 		$response = $this->graphql( compact( 'query' ) );
 		$expected = [
 			$this->expectedField( 'failingAuthConnection.nodes.0', self::NOT_NULL ),
```

Comment:
Per slack, I don't understand this entire test case. Since the test was previously passing, we should verify that we're testing the behavior we actually want.